Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM node:14.16.0
+ARG base_image=node:16-slim
 
-ENV ENV_NODE production
-ENV PORT 9900
-ENV NODE_ADDRESS 127.0.0.1
-ENV NODE_PORT 10001
-ENV NODE_TIMEOUT 10000
+FROM ${base_image}
+
+ENV ENV_NODE=production
+ENV PORT=9900
+ENV NODE_ADDRESS=172.17.0.1
```

Comment:
Why change the default to 172.17.0.1, or is the 17 a mistype?

Reply:
It's because that's the default IP of the host. I think this is the only reasonable default to use in a Docker context.